SimpleCam/SimpleCam/SimpleCam.m

```diff
@@ -303,6 +313,17 @@ - (void) setup {
     
     // -- PREPARE OUR CONTROLS -- //
     [self loadControls];
+    
+    // -- PREPARE CAMERA FLASH ANIMATION -- //
+    if (cameraCaptureFlashAnimation) {
+        [cameraCaptureFlashAnimation removeFromSuperview];
+        cameraCaptureFlashAnimation = nil;
+    }
+    
+    cameraCaptureFlashAnimation = [[UIView alloc] initWithFrame:self.view.frame];
```

Comment:
Let's make this `bounds` instead of frame, I know people have presented this control modally where it wasn't full screen, and this might throw it off.